README/ReleaseNotes/v624/index.md

```diff
@@ -66,6 +66,7 @@ See the discussion at [ROOT-11014](https://sft.its.cern.ch/jira/browse/ROOT-1101
 
 ## TTree Libraries
 
+- TTree now supports leaves of types `long` and `unsigned long` (and therefore also `std::size_t` on most systems). The corresponding leaflist letters are 'G' and 'g'.
```

Comment:
```suggestion
- TTree now supports for branch in "leave list mode" leaves of types `long` and `unsigned long` (and therefore also `std::size_t` on most systems). The corresponding leaflist letters are 'G' and 'g'.
```

Reply:
Hi @pcanal , thank you for taking a look. Can you please amend your suggestion with `s/for branch in "leave list mode"/, for branches in "leaflist mode",/`? :smile: